mjordan/islandora_workbench
# config refactoring

## Link to Github issue or other discussion

[Issue 320 i7Import refactoring](https://github.com/mjordan/islandora_workbench/issues/320)

## What does this PR do?

Externalizes config, provides sensible defaults, refactors some code.
## What changes were made?

> refactored get_islandora_7_content.py and added some optional parameters

## How to test / veryify this PR?

 > Modify sample config (or create your own) and run against an available i7 Instance.  The spreadsheet produced should be valid.
## Interested Parties

>  @mjordan

---

## Checklist
